.github/workflows/pull-request-management.yml

```diff
@@ -216,10 +216,10 @@ jobs:
         # Also test minimum ansible version for one scenario.
         include:
           - avd_scenario: 'eos_designs_unit_tests'
-            ansible_version: 'ansible-core==2.15.0'
+            ansible_version: 'ansible-core==2.16.0'
             pip_requirements: 'tmp-requirements-minimum.txt'
           - avd_scenario: 'eos_designs_unit_tests'
-            ansible_version: 'ansible-core>=2.15.0,<2.16.0 --upgrade'
+            ansible_version: 'ansible-core>=2.16.0,<2.17.0 --upgrade'
```

Comment:
We don't need this one, since it would just test 2.16.<latest>. We can remove this variant.

Reply:
update and removed workflow